org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/workspace/EclipseProjectConfig.xtend

```diff
@@ -66,8 +64,12 @@ class EclipseProjectConfig implements IProjectConfig {
 		emptySet
 	}
 
+	override getOutputFolders() {
```

Comment:
This would be a good default impl, if we decide to keep that method on the interface.

Reply:
Thanks. Right, but in favour for keeping the PR as small as possible, I removed the output folders again and work with another solution.